test/WebSites/TagHelpersWebSite/Views/Catalog_CacheTagHelper/_SplashPartial.cshtml

```diff
@@ -0,0 +1,5 @@
+﻿<partial-title>Listing items</partial-title>
+<cache asp-vary-by="@ViewBag.Category" data-content="some-content">
```

Comment:
It's meant to exhibit that the contents of the `cache` tag get cleared. I could be more explicit about it in a separate test.